docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -12,17 +12,18 @@ filter to be kept separate from any keys used by other systems. This document as
 will be to prefix the alias used by the filter with the word `KEK_`. If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
```

Comment:
I thought about alternative terms: "Key Admin", "KEK Admin", "Key Manager".. in the end I went back to Administrator.

Reply:
My only other suggestion would be `The Administrative Actor` 

but I think the `Role of the Administrator` is probably a better title.